### PR TITLE
Add example of `Dictionary::read()` and `write()`

### PR DESCRIPTION
**Pull request by vbkaisetsu**
See #122

### CODE REVIEW

**Review by kampersanda — COMMENTED** [review]
@vbkaisetsu 

I think that description is not needed in documentation of the `vibrato` crate because `vibrato::dictionary::Dictionary::read()` is not restricted to `zstd`, accepting `Read` objects.

"How we distribute our dictionaries" and "what CLI tools we provide" outside the vibrato crate should be kept separate from the API specification of the crate.

**Comment by vbkaisetsu** [comment]
@kampersanda Since the distributed dictionaries are compressed, and the CLI tool requires compressed ones, it would be necessary to explain that the API requires an uncompressed dictionary.

File: vibrato/src/dictionary.rs [hunk 1]
```diff
@@ -133,6 +133,10 @@ impl Dictionary {
 
     /// Creates a dictionary from a reader.
     ///
+    /// Unlike CLI tools, this function requires an uncompressed dictionary, so you need to use a
+    /// zstd decoder such as [zstd crate](https://docs.rs/zstd/latest/zstd/) or [ruzstd
+    /// crate](https://docs.rs/ruzstd/latest/ruzstd/) to decompress the dictionary beforehand.
```
Comment (kampersanda) [inline vibrato/src/dictionary.rs:138]:
```suggestion
    /// This function requires a dictionary in uncompressed format.
```
I would like to minimize the dependency between the `vibrate` crate and the outside as much as possible. The zstd format of distributed models is just one example and should not inherently be related to this API.

How about my suggestion?

**Comment by vbkaisetsu** [comment]
It needs more discussion

**Review by kampersanda — APPROVED** [review]
Thanks!

**Pull request merged**